storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/pkr-operation.cpp

```diff
@@ -85,49 +93,60 @@ RS_Status PKROperation::SetupTransaction() {
  * @return status
  */
 RS_Status PKROperation::SetupReadOperation() {
-  if (operations.size() != 0) {
-    return RS_CLIENT_ERROR(ERROR_006);
-  }
 
-  for (size_t i = 0; i < no_ops; i++) {
-    PKRRequest *req                        = requests[i];
-    const NdbDictionary::Table *table_dict = all_table_dicts[i];
-    NdbOperation *op                       = transaction->getNdbOperation(table_dict);
+  for (size_t opIdx = 0; opIdx < noOps; opIdx++) {
+    if (subOpTuples[opIdx].pkRequest->IsInvalidOp()) {
+      // this sub operation can not be processed
+          printf("Ignoring operation here1\n");
```

Comment:
Gather the 1 should not be there

Reply:
ahh, ok, it is for debugging, should you not have some way of logging for debug?

Reply:
my bad. this was for debugging. I have removed it